src/app/bug/bug.component.spec.ts

```diff
@@ -1,11 +1,12 @@
 import { ActivatedRoute } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
-import { createComponentFactory, Spectator, SpyObject } from '@ngneat/spectator';
+import { createComponentFactory, Spectator, SpectatorRouting, SpyObject } from '@ngneat/spectator';
 import { BugComponent } from './bug.component';
 import { BugService } from './shared/bug.service';
 import { Month } from './shared/models/month.model';
 import { Bug } from './shared/models/bug.model';
 import { BehaviorSubject, of } from 'rxjs';
+import * as faker from 'faker';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8845963)
```suggestion
import * as faker from "faker";
```